src/App.js

```diff
@@ -1,39 +1,64 @@
 import React, { Component } from 'react'
+import axios from 'axios'
 import Movie from './components/Movie'
+import Header from './components/Header'
 import MovieList from './components/MovieList'
 import SearchBar from './components/SearchBar'
-import movies from './movies.js'
+
+const DEFAULT_STATUS = 'Search a massive database of movies and TV shows!'
+const SearchStatus = ({text}) => (<h1>{text}</h1>)
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies,
-      searchField: ''
+      movies: [],
+      search: '',
+      status: DEFAULT_STATUS
     }
-    this.onSearchChange = this.onSearchChange.bind(this)
+    this.handleSearchChange = this.handleSearchChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
   }
 
-  onSearchChange(e){
-    this.setState({ searchField: e.target.value })
+  fetchMovies(search){
+    const { movies, status } = this.state
+    let typedSearch = `http://www.omdbapi.com/?apikey=df1379a2&s=${search}`
+    axios.get(typedSearch)
+    .then(result=> {
```

Comment:
This is a little hard to read. Also you don't need to return, you're just setting state based on the outcome.

I would do:
```
.then(({ data }) => {
  const status = data.Error || ''
  const movies = data.Search || []
  this.setState({ status, movies })
})
```

Reply:
done!